app/src/main/kotlin/at/bitfire/davdroid/webdav/WebDavMountRepository.kt

```diff
@@ -114,7 +115,7 @@ class WebDavMountRepository @Inject constructor(
     internal suspend fun hasWebDav(
```

Comment:
Looks good, but KDoc of what is returned under which conditions would be nice (as it's return value is not obvious from the method name).